<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>externaldownloader</class>
 <widget class="QWidget" name="externaldownloader">
  <property name="styleSheet">
   <string notr="true">
      QWidget {
          background-color: #2b2b2b;
          color: #bbbbbb; /* Default text color */
      }
      QLabel {
          color: #bbbbbb;
      }
      QLineEdit, QTextEdit, QPlainTextEdit {
          background-color: #3c3f41;
          color: #bbbbbb;
          border: 1px solid #555555;
          selection-background-color: #007acc;
          selection-color: white;
      }
      QPushButton, QToolButton {
          background-color: #555555;
          color: #bbbbbb;
          border: 1px solid #666666;
          padding: 5px;
      }
      QPushButton:hover, QToolButton:hover {
          background-color: #666666;
      }
      QPushButton:pressed, QToolButton:pressed {
          background-color: #444444;
      }
      QComboBox {
          background-color: #3c3f41;
          color: #bbbbbb;
          border: 1px solid #555555;
          selection-background-color: #007acc;
          /* selection-color: white; */
      }
      QComboBox::drop-down {
          border: none;
      }
      QComboBox QAbstractItemView {
          background-color: #3c3f41;
          color: #bbbbbb;
          selection-background-color: #007acc;
          selection-color: white;
          border: 1px solid #555555;
          outline: 0px;
      }
      QDateEdit {
          background-color: #3c3f41;
          color: #bbbbbb;
          border: 1px solid #555555;
          selection-background-color: #007acc;
          selection-color: white;
      }
      QDateEdit::drop-down {
           /* Style dropdown button if needed */
      }
      QDateEdit QAbstractItemView {
          background-color: #3c3f41;
          color: #bbbbbb;
          selection-background-color: #007acc;
      }
      QTableView {
          background-color: #3c3f41;
          color: #bbbbbb;
          gridline-color: #555555;
          selection-background-color: #007acc;
          selection-color: white;
      }
      QHeaderView::section {
          background-color: #555555;
          color: #bbbbbb;
          padding: 4px;
          border: 1px solid #666666;
      }
      QProgressBar {
          border: 1px solid #555555;
          text-align: center;
          color: #bbbbbb;
          background-color: #3c3f41;
      }
      QProgressBar::chunk {
          background-color: #007acc;
          width: 10px; 
          margin: 0.5px;
      }
     </string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>External Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/48x48.png</normaloff>:/48x48.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FFF;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Chancery L</family>
     <pointsize>17</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>External Downloader</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_teminal_yd1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>671</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: rgb(83, 28, 28);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="tabStopDistance">
    <double>80</double>
   </property>
   <property name="cursorWidth">
    <number>2</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame_loading">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>671</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(92, 80, 80);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Palladio L</family>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Moving mountains for you... please wait while our little hamsters are doing the work ;-)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Palladio L</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFF</string>
    </property>
    <property name="text">
     <string>Dowloading and Converting Video...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Palladio L</family>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Even with a lot of electrons one must persist on thinking positive!</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre 1:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_g2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-color: rgb(87, 118, 255);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_g1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-color: rgb(87, 118, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_song">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_artist">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_cplp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Check this to indicate that this is a Portuguese speaking track</string>
   </property>
   <property name="text">
    <string>CPLP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Song:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Artist(s):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Genre 2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Published date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_publishedDate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Video Link:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_videoLink">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Manage Genres</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/icons/icon48x48.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="bt_youtube_getIt">
      <property name="text">
       <string>Get it!</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/svn-update.png</normaloff>:/icons/svn-update.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_clear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_close">
      <property name="text">
       <string>Close</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
